### DISCUSSION TITLE
[@types/react] datepicker missing  -> types for  excludeDateIntervals

### DISCUSSION THREAD

**Discussion by Kristaps006** [DefinitelyTyped/DefinitelyTyped#59764] [Issues with a @types package]
`  Property 'excludeDateIntervals' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<ReactDatePicker`

It seems like ` excludeDateIntervals ` doesn't have a type

Thank you

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
I assume you meant the package `react-datepicker`?
I've just opened #59767

**Reply by Kristaps006** [reply]
Yes, sorry 👍  I did
